src/Test.FunctionalTests.BinSkim.Driver/CommandLineTests.cs

```diff
@@ -70,5 +70,40 @@ public void MostlyFunctionlessCommandlineTest()
             builder.Length.Should().Be(0,
                 $"all test cases should pass, but the following test cases failed:\n{builder}");
         }
+
+        [Fact]
+        public void DisableTelemetryCommandlineTest()
```

Comment:
I don't know what the rest of this suite looks like, and I'm not 100% sure I'm following *this* test.   So tell me if this is here and I'm not seeing it.  But: I think you need a test that disables telemetry, runs a scan against a well-known file (I see this much) and then verifies that the scan results are the same as if you had not passed in disable-telemetry (I don't see this part.)

Reply:
good input. Added another test.